packages/create-cloudflare/src/frameworks/solid/index.ts

```diff
@@ -18,31 +16,20 @@ const generate = async (ctx: PagesGeneratorContext) => {
 const configure = async (ctx: PagesGeneratorContext) => {
 	process.chdir(ctx.project.path);
 
-	// Install the pages adapter
-	const pkg = "solid-start-cloudflare-pages";
-	await installPackages([pkg], {
-		dev: true,
-		startText: "Adding the Cloudflare Pages adapter",
-		doneText: `${brandColor(`installed`)} ${dim(pkg)}`,
-	});
-
-	// modify the vite config
-	const viteConfigPath = usesTypescript()
-		? `./vite.config.ts`
-		: `./vite.config.js`;
-	writeFile(viteConfigPath, viteConfig);
-	updateStatus(
-		`Adding the Cloudflare Pages adapter to ${blue(viteConfigPath)}`
-	);
+	// Note: we should update the vite.config.ts/js file here and set the preset on
+	// start.server.preset (as described in the solidStart docs: https://start.solidjs.com/api/vite#configuring-your-application)
+	// but that doesn't seem to work, so for now we just set the PRESET env variable in the build script
+	// later we should amend this when it's properly handled on solidStart's side
```

Comment:
never mind, it actually does work, the config needs to look like this:
```
import { defineConfig } from "@solidjs/start/config";

export default defineConfig({
  start: {
    server: {
      preset: "cloudflare-pages"
    }
  }
});
```
(PS: I am pretty sure this didn't work when I opened the PR, maybe they've fixed it upstream 🤷 )